<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BORSE-PC" ignored="69" name="Default test" tests="192" failures="26" timestamp="2020-02-06T21:05:39 IST" time="26909.588" errors="0">
  <testcase name="AddAlreadyUsedNVRManuallyTest" time="14.992" classname="testcases.NVRsTest"/>
  <testcase name="AddGroupAndCheckInMonitoringTest" time="62.991" classname="testcases.GroupsHierarchyTest"/>
  <testcase name="AddNVRAndCheckStatusVTest" time="40.734" classname="testcases.NVRsTest"/>
  <testcase name="AddNVRAndPressApplyTest" time="37.629" classname="testcases.NVRsTest"/>
  <testcase name="AddNVRManuallyAndPressApplyTest" time="34.86" classname="testcases.NVRsTest"/>
  <testcase name="AddNVRManuallyAndPressCloseTest" time="16.067" classname="testcases.NVRsTest"/>
  <testcase name="AddNVRManuallyAndSwitchONSecuredTest" time="50.02" classname="testcases.NVRsTest"/>
  <testcase name="AddNVRManuallyCheckStatusTest" time="30.494" classname="testcases.NVRsTest"/>
  <testcase name="AddNVRManuallyWithSecuredPortAndSwitchONSecuredTest" time="89.599" classname="testcases.NVRsTest"/>
  <testcase name="AddNVRManuallyWithSecuredPortTest" time="197.331" classname="testcases.NVRsTest">
    <failure type="java.lang.AssertionError" message="expected [Communication Failure] but found [Connecting...]">
      <![CDATA[java.lang.AssertionError: expected [Communication Failure] but found [Connecting...]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.NVRsTest.AddNVRManuallyWithSecuredPortTest(NVRsTest.java:812)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- AddNVRManuallyWithSecuredPortTest -->
  <testcase name="AddNVRWithSecuredPortTest" time="99.295" classname="testcases.NVRsTest"/>
  <testcase name="AddNewTourAndCancelTest" time="6.993" classname="testcases.ToursElementsTest"/>
  <testcase name="AddNewTourExitAndCancelTest" time="21.473" classname="testcases.ToursElementsTest"/>
  <testcase name="AddNewTourExitAndDoNotSaveTest" time="31.953" classname="testcases.ToursElementsTest"/>
  <testcase name="AddNewTourExitAndSaveTest" time="32.254" classname="testcases.ToursElementsTest"/>
  <testcase name="AddNewTourTest" time="10.953" classname="testcases.ToursTest"/>
  <testcase name="AddNewTourWithAllParametersTest" time="113.209" classname="testcases.ToursTest"/>
  <testcase name="AddNewTourWithoutViewAndCancelTest" time="5.204" classname="testcases.ToursElementsTest"/>
  <testcase name="AddNewTourWithoutViewExitAndCancelTest" time="16.362" classname="testcases.ToursElementsTest"/>
  <testcase name="AddNewTourWithoutViewExitAndDoNotSaveTest" time="26.133" classname="testcases.ToursElementsTest"/>
  <testcase name="AddNewTourWithoutViewExitAndSaveTest" time="4.054" classname="testcases.ToursElementsTest"/>
  <testcase name="AddNewViewAndPressCancelTest" time="8.075" classname="testcases.ViewsElementsTest"/>
  <testcase name="AddNewViewExitAndPressCancelTest" time="20.615" classname="testcases.ViewsElementsTest"/>
  <testcase name="AddNewViewExitAndPressDoNotSaveTest" time="26.269" classname="testcases.ViewsElementsTest"/>
  <testcase name="AddNewViewExitAndPressSaveTest" time="26.297" classname="testcases.ViewsElementsTest"/>
  <testcase name="AddNewViewWithLayoutExitAndCancelTest" time="18.038" classname="testcases.ViewsElementsTest"/>
  <testcase name="AddNewViewWithLayoutExitAndDoNotSaveTest" time="24.448" classname="testcases.ViewsElementsTest"/>
  <testcase name="AddNewViewWithLayoutExitAndSaveTest" time="3.826" classname="testcases.ViewsElementsTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testcases.ViewsElementsTest.AddNewViewWithLayoutExitAndSaveTest(ViewsElementsTest.java:266)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- AddNewViewWithLayoutExitAndSaveTest -->
  <testcase name="AddNewViewWithoutLayoutAndExitAndCancelTest" time="17.213" classname="testcases.ViewsElementsTest"/>
  <testcase name="AddNewViewWithoutLayoutAndExitAndDoNotSaveTest" time="24.112" classname="testcases.ViewsElementsTest"/>
  <testcase name="AddNewViewWithoutLayoutAndExitAndSaveTest" time="17.52" classname="testcases.ViewsElementsTest"/>
  <testcase name="AddNewWebPageTest" time="0.502" classname="testcases.WebPagesTest"/>
  <testcase name="AddSubGroupTest" time="36.354" classname="testcases.GroupsHierarchyTest"/>
  <testcase name="AddViewAndCancelTest" time="14.809" classname="testcases.ToursElementsTest"/>
  <testcase name="AddViewExitAndCancelTest" time="22.891" classname="testcases.ToursElementsTest"/>
  <testcase name="AddViewExitAndDoNotSaveTest" time="40.861" classname="testcases.ToursElementsTest"/>
  <testcase name="AddViewExitAndSaveTest" time="42.369" classname="testcases.ToursElementsTest"/>
  <testcase name="AddViewTest" time="34.154" classname="testcases.ToursTest"/>
  <testcase name="CancelRemovalNVRsByButtonTest" time="11.373" classname="testcases.NVRsTest"/>
  <testcase name="CancelRemovalNVRsByIconTest" time="11.523" classname="testcases.NVRsTest"/>
  <testcase name="CancelWebPageRemovingByButtonTest" time="31.728" classname="testcases.WebPagesTest"/>
  <testcase name="CancelWebPageRemovingByIconTest" time="32.426" classname="testcases.WebPagesTest"/>
  <testcase name="ChangeIPAdressAndCheckStatusTest" time="9.035" classname="testcases.NVRsTest">
    <failure type="java.lang.AssertionError" message="expected [172.20.1.46IP] but found [IP]">
      <![CDATA[java.lang.AssertionError: expected [172.20.1.46IP] but found [IP]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.NVRsTest.ChangeIPAdressAndCheckStatusTest(NVRsTest.java:1356)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangeIPAdressAndCheckStatusTest -->
  <testcase name="ChangeIPAdressExitCancelUnsavedChangesTest" time="189.1" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="expected [172.20.1.46IP10] but found [IP10]">
      <![CDATA[java.lang.AssertionError: expected [172.20.1.46IP10] but found [IP10]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.NVRsElementsTest.ChangeIPAdressExitCancelUnsavedChangesTest(NVRsElementsTest.java:377)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangeIPAdressExitCancelUnsavedChangesTest -->
  <testcase name="ChangeIPAdressExitPressDontSaveUnsavedChangesTest" time="243.232" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangeIPAdressPressCancelExitTest" time="208.575" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangeLayoutAndPressCancelTest" time="25.393" classname="testcases.ViewsElementsTest">
    <failure type="java.lang.AssertionError" message="expected [9] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [9] but found [0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at testcases.ViewsElementsTest.ChangeLayoutAndPressCancelTest(ViewsElementsTest.java:1331)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangeLayoutAndPressCancelTest -->
  <testcase name="ChangeLayoutExitAndPressCancelTest" time="37.598" classname="testcases.ViewsElementsTest">
    <failure type="java.lang.AssertionError" message="expected [4] but found [9]">
      <![CDATA[java.lang.AssertionError: expected [4] but found [9]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at testcases.ViewsElementsTest.ChangeLayoutExitAndPressCancelTest(ViewsElementsTest.java:1403)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangeLayoutExitAndPressCancelTest -->
  <testcase name="ChangeLayoutExitAndPressDontSaveTest" time="45.918" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeLayoutExitAndPressSaveTest" time="103.788" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeNameAndCancelTest" time="5.357" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeNameAndPressCancelTest" time="69.528" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeNameExitAndCancelTest" time="20.68" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeNameExitAndDoNotSaveTest" time="36.476" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeNameExitAndPressCancelTest" time="37.556" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeNameExitAndPressDoNotSaveTest" time="28.531" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeNameExitAndPressSaveTest" time="85.567" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeNameExitAndSaveTest" time="35.516" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeNameExitPressDontSaveUnsavedChangesTest" time="70.105" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangeNameExitPressSaveUnsavedChangesTest" time="244.4" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangeNamePressCancelTest" time="6.932" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangeNameTest" time="22.835" classname="testcases.NVRsTest"/>
  <testcase name="ChangeNameTest" time="38.083" classname="testcases.ToursTest"/>
  <testcase name="ChangeNameTest" time="38.851" classname="testcases.WebPagesTest"/>
  <testcase name="ChangeOrderForViewTest" time="3.225" classname="testcases.ToursTest"/>
  <testcase name="ChangePasswordToInvalidAndCheckStatusTest" time="193.214" classname="testcases.NVRsTest"/>
  <testcase name="ChangePortAndCheckStatusTest" time="145.396" classname="testcases.NVRsTest"/>
  <testcase name="ChangePortExitCancelUnsavedChangesTest" time="22.502" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangePortExitPressDontSaveUnsavedChangesTest" time="219.511" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangePortExitPressSaveUnsavedChangesTest" time="217.623" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangePortPressCancelTest" time="17.791" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="expected [80] but found [548]">
      <![CDATA[java.lang.AssertionError: expected [80] but found [548]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.NVRsElementsTest.ChangePortPressCancelTest(NVRsElementsTest.java:557)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangePortPressCancelTest -->
  <testcase name="ChangeRefreshTimeStatusTest" time="25.665" classname="testcases.WebPagesTest"/>
  <testcase name="ChangeRefreshTimeValueBySpinnerTest" time="10.946" classname="testcases.WebPagesTest"/>
  <testcase name="ChangeRefreshTimeValueTest" time="26.363" classname="testcases.WebPagesTest"/>
  <testcase name="ChangeRemarksAndCancelTest" time="5.888" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeRemarksAndPressCancelTest" time="56.085" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeRemarksExitAndCancelTest" time="25.221" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeRemarksExitAndDoNotSaveTest" time="42.486" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeRemarksExitAndPressCancelTest" time="24.965" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeRemarksExitAndPressDoNotSaveTest" time="73.041" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeRemarksExitAndPressSaveTest" time="25.999" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeRemarksExitAndSaveTest" time="45.969" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeRemarksTest" time="28.208" classname="testcases.ToursTest"/>
  <testcase name="ChangeRemarksTest" time="24.947" classname="testcases.WebPagesTest"/>
  <testcase name="ChangeSecuredAndCheckStatusTest" time="241.527" classname="testcases.NVRsTest">
    <failure type="java.lang.AssertionError" message="expected [V] but found [NVR cannot be synced. Its system unique id belongs to another NVR.]">
      <![CDATA[java.lang.AssertionError: expected [V] but found [NVR cannot be synced. Its system unique id belongs to another NVR.]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.NVRsTest.ChangeSecuredAndCheckStatusTest(NVRsTest.java:1533)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangeSecuredAndCheckStatusTest -->
  <testcase name="ChangeURLTest" time="22.702" classname="testcases.WebPagesTest"/>
  <testcase name="ChangeUserNameAndPressCancel" time="174.455" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangeUserNameExitCancelUnsavedChangesTest" time="202.983" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangeUserNameExitPressDontSaveUnsavedChangesTest" time="213.278" classname="testcases.NVRsElementsTest"/>
  <testcase name="ChangeUserNameExitPressSaveUnsavedChangesTest" time="211.59" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="expected [UserName] but found [ADMIN]">
      <![CDATA[java.lang.AssertionError: expected [UserName] but found [ADMIN]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.NVRsElementsTest.ChangeUserNameExitPressSaveUnsavedChangesTest(NVRsElementsTest.java:1083)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangeUserNameExitPressSaveUnsavedChangesTest -->
  <testcase name="ChangeUserNameToInvalidAndCheckStatusTest" time="186.961" classname="testcases.NVRsTest"/>
  <testcase name="ChangeViewTimeAndCancelTest" time="6.372" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeViewTimeBySpinnersAndCancelTest" time="25.63" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeViewTimeBySpinnersExitAndCancelTest" time="6.168" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeViewTimeBySpinnersExitAndDoNotSaveTest" time="53.117" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeViewTimeBySpinnersExitAndSaveTest" time="226.884" classname="testcases.ToursElementsTest">
    <failure type="org.openqa.selenium.NoSuchElementException" message="Unable to find element with xpath == //div[@class = &amp;apos;modal-footer-confirm&amp;apos;]/button[contains(text(), &amp;apos;Save&amp;apos;)] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &amp;apos;3.9.1&amp;apos;, revision: &amp;apos;63f7b50&amp;apos;, time: &amp;apos;2018-02-07T22:25:02.294Z&amp;apos;
System info: host: &amp;apos;BORSE-PC&amp;apos;, ip: &amp;apos;172.20.1.42&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_202&amp;apos;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:25358/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: cd37a518-f5b5-425e-8161-66ce39571d04
*** Element info: {Using=xpath, value=//div[@class = &amp;apos;modal-footer-confirm&amp;apos;]/button[contains(text(), &amp;apos;Save&amp;apos;)]}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to find element with xpath == //div[@class = 'modal-footer-confirm']/button[contains(text(), 'Save')] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'BORSE-PC', ip: '172.20.1.42', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:25358/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: cd37a518-f5b5-425e-8161-66ce39571d04
*** Element info: {Using=xpath, value=//div[@class = 'modal-footer-confirm']/button[contains(text(), 'Save')]}
at sun.reflect.GeneratedConstructorAccessor35.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:160)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:371)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy14.isEnabled(Unknown Source)
at Utilities.Page.SaveUnsavedChangesButtonIsEnabled(Page.java:369)
at testcases.ToursElementsTest.ChangeViewTimeBySpinnersExitAndSaveTest(ToursElementsTest.java:1782)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangeViewTimeBySpinnersExitAndSaveTest -->
  <testcase name="ChangeViewTimeBySpinnersTest" time="45.458" classname="testcases.ToursTest"/>
  <testcase name="ChangeViewTimeExitAndCancelTest" time="23.081" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeViewTimeExitAndDoNotSaveTest" time="36.575" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeViewTimeExitAndSaveTest" time="6.183" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeViewTimeTest" time="6.138" classname="testcases.ToursTest"/>
  <testcase name="ChangeViewTimeToLessThanMinValueTest" time="6.078" classname="testcases.ToursTest"/>
  <testcase name="ChangeViewTimeToMoreThanMaxValueTest" time="6.18" classname="testcases.ToursTest"/>
  <testcase name="ChangeVisibilityStatusAndCancelTest" time="3.542" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeVisibilityStatusAndPressCancelTest" time="29.212" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeVisibilityStatusExitAndCancelTest" time="23.644" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeVisibilityStatusExitAndDoNotSaveTest" time="30.356" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeVisibilityStatusExitAndPressCancelTest" time="92.849" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeVisibilityStatusExitAndPressDoNotSaveTest" time="37.241" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeVisibilityStatusExitAndPressSaveTest" time="33.904" classname="testcases.ViewsElementsTest"/>
  <testcase name="ChangeVisibilityStatusExitAndSaveTest" time="6.324" classname="testcases.ToursElementsTest"/>
  <testcase name="ChangeVisibilityStatusTest" time="5.854" classname="testcases.ToursTest"/>
  <testcase name="ChangeVisibilityStatusTest" time="43.251" classname="testcases.WebPagesTest"/>
  <testcase name="CheckPropertiesAfterChangeIPAdressTest" time="51.901" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="expected [] but found [172.20.1.42 Name]">
      <![CDATA[java.lang.AssertionError: expected [] but found [172.20.1.42 Name]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.NVRsElementsTest.CheckPropertiesAfterChangeIPAdressTest(NVRsElementsTest.java:522)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CheckPropertiesAfterChangeIPAdressTest -->
  <testcase name="CheckPropertiesAfterChangeNameTest" time="50.659" classname="testcases.NVRsElementsTest"/>
  <testcase name="CheckPropertiesAfterChangePasswordTest" time="59.635" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="expected [UserName] but found [ADMIN]">
      <![CDATA[java.lang.AssertionError: expected [UserName] but found [ADMIN]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.NVRsElementsTest.CheckPropertiesAfterChangePasswordTest(NVRsElementsTest.java:1175)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CheckPropertiesAfterChangePasswordTest -->
  <testcase name="CheckPropertiesAfterChangePortTest" time="68.817" classname="testcases.NVRsElementsTest"/>
  <testcase name="CheckPropertiesAfterChangeUserNameTest" time="58.025" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="expected [172.20.1.46] but found [172.20.1.42 Name]">
      <![CDATA[java.lang.AssertionError: expected [172.20.1.46] but found [172.20.1.42 Name]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.NVRsElementsTest.CheckPropertiesAfterChangeUserNameTest(NVRsElementsTest.java:1126)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CheckPropertiesAfterChangeUserNameTest -->
  <testcase name="ClearViewAndPressCancelTest" time="11.5" classname="testcases.ViewsElementsTest">
    <failure type="java.lang.AssertionError" message="expected [1] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at testcases.ViewsElementsTest.ClearViewAndPressCancelTest(ViewsElementsTest.java:1107)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ClearViewAndPressCancelTest -->
  <testcase name="ClearViewExitAndPressCancelTest" time="96.48" classname="testcases.ViewsElementsTest"/>
  <testcase name="ClearViewExitAndPressDoNotSaveTest" time="37.749" classname="testcases.ViewsElementsTest"/>
  <testcase name="ClearViewExitAndPressSaveTest" time="26.077" classname="testcases.ViewsElementsTest"/>
  <testcase name="DecreasePortExitCancelUnsavedChangesTest" time="10.969" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testcases.NVRsElementsTest.DecreasePortExitCancelUnsavedChangesTest(NVRsElementsTest.java:811)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- DecreasePortExitCancelUnsavedChangesTest -->
  <testcase name="DecreasePortExitPressSaveUnsavedChangesTest" time="185.822" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testcases.NVRsElementsTest.DecreasePortExitPressSaveUnsavedChangesTest(NVRsElementsTest.java:901)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- DecreasePortExitPressSaveUnsavedChangesTest -->
  <testcase name="DeleteAllViewsAndCancelTest" time="6.124" classname="testcases.ToursElementsTest"/>
  <testcase name="DeleteAllViewsExitAndCancelTest" time="22.053" classname="testcases.ToursElementsTest"/>
  <testcase name="DeleteAllViewsExitAndDoNotSaveTest" time="27.301" classname="testcases.ToursElementsTest"/>
  <testcase name="DeleteAllViewsExitAndSaveTest" time="3.213" classname="testcases.ToursElementsTest"/>
  <testcase name="DeleteAllViewsTest" time="1.847" classname="testcases.ToursTest"/>
  <testcase name="DeleteGroupAndCheckInMonitoringTest" time="74.735" classname="testcases.GroupsHierarchyTest"/>
  <testcase name="DeleteGroupAndPressNoTest" time="32.266" classname="testcases.GroupsHierarchyTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testcases.GroupsHierarchyTest.DeleteGroupAndPressNoTest(GroupsHierarchyTest.java:500)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- DeleteGroupAndPressNoTest -->
  <testcase name="DeleteOneNVRClickOnAnotherNVRAndCheckPropertiesPanelTest" time="21.83" classname="testcases.NVRsTest"/>
  <testcase name="DeleteResourceByButtonAndCancelRemovingTest" time="19.269" classname="testcases.GroupsHierarchyTest"/>
  <testcase name="DeleteResourceByButtonAndCheckInMonitoringTest" time="47.476" classname="testcases.GroupsHierarchyTest"/>
  <testcase name="DeleteTourByButtonAndPressNoTest" time="25.831" classname="testcases.ToursTest"/>
  <testcase name="DeleteTourByButtonTest" time="29.275" classname="testcases.ToursTest"/>
  <testcase name="DeleteTourByIconAndPressNoTest" time="18.457" classname="testcases.ToursTest"/>
  <testcase name="DeleteTourByIconTest" time="29.296" classname="testcases.ToursTest"/>
  <testcase name="DeleteURLTest" time="6.304" classname="testcases.WebPagesTest"/>
  <testcase name="DeleteViewAndCancelTest" time="9.077" classname="testcases.ToursElementsTest"/>
  <testcase name="DeleteViewExitAndCancelTest" time="0.345" classname="testcases.ToursElementsTest"/>
  <testcase name="DeleteViewExitAndDoNotSaveTest" time="25.577" classname="testcases.ToursElementsTest"/>
  <testcase name="DeleteViewExitAndSaveTest" time="1.513" classname="testcases.ToursElementsTest"/>
  <testcase name="DeleteViewFromViewsPageTest" time="58.987" classname="testcases.ToursTest"/>
  <testcase name="DeleteViewTest" time="35.521" classname="testcases.ToursTest"/>
  <testcase name="DeleteWebPageByButtonTest" time="50.688" classname="testcases.WebPagesTest"/>
  <testcase name="DeleteWebPageByIconTest" time="45.723" classname="testcases.WebPagesTest"/>
  <testcase name="DragResourceToGroupTest" time="44.435" classname="testcases.GroupsHierarchyTest"/>
  <testcase name="FilterDiscoveredNVRsTest" time="15.441" classname="testcases.NVRsTest"/>
  <testcase name="FilterNVRsTest" time="3.219" classname="testcases.NVRsTest"/>
  <testcase name="FilterToursTest" time="2.686" classname="testcases.ToursTest"/>
  <testcase name="FilterViewsTest" time="4.8" classname="testcases.ToursTest"/>
  <testcase name="IncreasePortExitPressDontSaveUnsavedChangesTest" time="166.733" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testcases.NVRsElementsTest.IncreasePortExitPressDontSaveUnsavedChangesTest(NVRsElementsTest.java:851)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- IncreasePortExitPressDontSaveUnsavedChangesTest -->
  <testcase name="InputInvalidIPAdressInAddNVRManualyTest" time="20.155" classname="testcases.NVRsTest"/>
  <testcase name="InputInvalidPasswordInAddNVRManualyTest" time="39.763" classname="testcases.NVRsTest"/>
  <testcase name="InputInvalidPortInAddNVRManualyTest" time="47.909" classname="testcases.NVRsTest"/>
  <testcase name="InputInvalidUserNameInAddNVRManualyTest" time="38.0" classname="testcases.NVRsTest"/>
  <testcase name="MultipleNVRsAdditingTest" time="75.156" classname="testcases.NVRsTest"/>
  <testcase name="NewTourWithEmptyViewTest" time="17.992" classname="testcases.ToursTest"/>
  <testcase name="RemoveNVRsByIconTest" time="21.305" classname="testcases.NVRsTest"/>
  <testcase name="RenameGroupAndCheckInMonitoringTest" time="42.672" classname="testcases.GroupsHierarchyTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [false] but found [true]">
      <![CDATA[java.lang.AssertionError: did not expect to find [false] but found [true]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at testcases.GroupsHierarchyTest.RenameGroupAndCheckInMonitoringTest(GroupsHierarchyTest.java:307)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- RenameGroupAndCheckInMonitoringTest -->
  <testcase name="RenameGroupAndClickCancelIconTest" time="4.626" classname="testcases.GroupsHierarchyTest"/>
  <testcase name="ResourcesFilterTest" time="4.742" classname="testcases.GroupsHierarchyTest"/>
  <testcase name="AddNewViewTest" time="62.538" classname="testcases.ViewsTest"/>
  <testcase name="AddNewViewWithoutLayoutTest" time="5.826" classname="testcases.ViewsTest"/>
  <testcase name="CancelDeletingViewByButton" time="33.038" classname="testcases.ViewsTest"/>
  <testcase name="CancelRemovingViewByIconTest" time="32.105" classname="testcases.ViewsTest"/>
  <testcase name="ChangeLayoutTest" time="99.177" classname="testcases.ViewsTest"/>
  <testcase name="ChangeNameTest" time="102.54" classname="testcases.ViewsTest">
    <failure type="org.openqa.selenium.NoSuchElementException" message="Unable to find element with xpath == //div[@class=&amp;apos;ui-grid-canvas&amp;apos;]//div[@title=&amp;apos;new view name54057&amp;apos;] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &amp;apos;3.9.1&amp;apos;, revision: &amp;apos;63f7b50&amp;apos;, time: &amp;apos;2018-02-07T22:25:02.294Z&amp;apos;
System info: host: &amp;apos;BORSE-PC&amp;apos;, ip: &amp;apos;172.20.1.42&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_202&amp;apos;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:10526/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: 1285eb69-9312-4eaa-b2ee-a8ee8a99c7fa
*** Element info: {Using=xpath, value=//div[@class=&amp;apos;ui-grid-canvas&amp;apos;]//div[@title=&amp;apos;new view name54057&amp;apos;]}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to find element with xpath == //div[@class='ui-grid-canvas']//div[@title='new view name54057'] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'BORSE-PC', ip: '172.20.1.42', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:10526/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: 1285eb69-9312-4eaa-b2ee-a8ee8a99c7fa
*** Element info: {Using=xpath, value=//div[@class='ui-grid-canvas']//div[@title='new view name54057']}
at sun.reflect.GeneratedConstructorAccessor35.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:160)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:371)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
at pageObjects.ViewsPage.ClickOnViewByName(ViewsPage.java:378)
at testcases.ViewsTest.ChangeNameTest(ViewsTest.java:283)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangeNameTest -->
  <testcase name="ChangeRemarksTest" time="85.177" classname="testcases.ViewsTest"/>
  <testcase name="ChangeVisibilityStatusTest" time="84.487" classname="testcases.ViewsTest"/>
  <testcase name="ClearViewAndDragResourceTest" time="68.185" classname="testcases.ViewsTest"/>
  <testcase name="ClearViewTest" time="15.633" classname="testcases.ViewsTest"/>
  <testcase name="DeleteViewByButtonTest" time="45.977" classname="testcases.ViewsTest"/>
  <testcase name="DeleteViewByIconTest" time="41.198" classname="testcases.ViewsTest"/>
  <testcase name="ResourceFilterTest" time="7.843" classname="testcases.ViewsTest"/>
  <testcase name="SelectNVRandPressCancelTest" time="24.102" classname="testcases.NVRsTest"/>
  <testcase name="SortingDiscoveredNVRsTest" time="20.971" classname="testcases.NVRsTest"/>
  <testcase name="SortingNVRsByIPAdressTest" time="115.204" classname="testcases.NVRsTest">
    <failure type="org.openqa.selenium.JavascriptException" message="JavaScript error (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: &amp;apos;3.9.1&amp;apos;, revision: &amp;apos;63f7b50&amp;apos;, time: &amp;apos;2018-02-07T22:25:02.294Z&amp;apos;
System info: host: &amp;apos;BORSE-PC&amp;apos;, ip: &amp;apos;172.20.1.42&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_202&amp;apos;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:37701/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: b27461fa-f68d-4136-b057-de9b80a0a441">
      <![CDATA[org.openqa.selenium.JavascriptException: JavaScript error (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'BORSE-PC', ip: '172.20.1.42', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:37701/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: b27461fa-f68d-4136-b057-de9b80a0a441
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:160)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:537)
at Utilities.Page.JavaScriptClick(Page.java:757)
at pageObjects.NVRsPage.AddDiscoveredNVRs(NVRsPage.java:934)
at pageObjects.NVRsPage.IfNVRIsNotExistAddIt(NVRsPage.java:955)
at testcases.NVRsTest.SortingNVRsByIPAdressTest(NVRsTest.java:1226)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SortingNVRsByIPAdressTest -->
  <testcase name="SortingNVRsByNameTest" time="41.686" classname="testcases.NVRsTest">
    <failure type="org.openqa.selenium.JavascriptException" message="JavaScript error (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: &amp;apos;3.9.1&amp;apos;, revision: &amp;apos;63f7b50&amp;apos;, time: &amp;apos;2018-02-07T22:25:02.294Z&amp;apos;
System info: host: &amp;apos;BORSE-PC&amp;apos;, ip: &amp;apos;172.20.1.42&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_202&amp;apos;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:15575/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: ba232456-6bd3-403d-92e7-f1e7dfe2392a">
      <![CDATA[org.openqa.selenium.JavascriptException: JavaScript error (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'BORSE-PC', ip: '172.20.1.42', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:15575/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: ba232456-6bd3-403d-92e7-f1e7dfe2392a
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:160)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:537)
at Utilities.Page.JavaScriptClick(Page.java:757)
at pageObjects.NVRsPage.AddDiscoveredNVRs(NVRsPage.java:934)
at pageObjects.NVRsPage.IfNVRIsNotExistAddIt(NVRsPage.java:955)
at testcases.NVRsTest.SortingNVRsByNameTest(NVRsTest.java:1250)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SortingNVRsByNameTest -->
  <testcase name="SortingNVRsByVersionTest" time="7.794" classname="testcases.NVRsTest"/>
  <testcase name="SwitchOffStorageDefinitionsAndPressNoTest" time="19.697" classname="testcases.NVRsTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testcases.NVRsTest.SwitchOffStorageDefinitionsAndPressNoTest(NVRsTest.java:1775)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SwitchOffStorageDefinitionsAndPressNoTest -->
  <testcase name="SwitchStorageDefinitionsAndPressCancelTest" time="9.611" classname="testcases.NVRsElementsTest"/>
  <testcase name="SwitchStorageDefinitionsExitAndPressCancelTest" time="24.238" classname="testcases.NVRsElementsTest"/>
  <testcase name="SwitchStorageDefinitionsExitAndPressDontSaveButtonTest" time="47.373" classname="testcases.NVRsElementsTest"/>
  <testcase name="SwitchStorageDefinitionsExitAndPressSaveButtonTest" time="50.538" classname="testcases.NVRsElementsTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testcases.NVRsElementsTest.SwitchStorageDefinitionsExitAndPressSaveButtonTest(NVRsElementsTest.java:1578)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SwitchStorageDefinitionsExitAndPressSaveButtonTest -->
  <testcase name="SwitchStorageDefinitionsTest" time="11.086" classname="testcases.NVRsTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [false] but found [true]">
      <![CDATA[java.lang.AssertionError: did not expect to find [false] but found [true]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at testcases.NVRsTest.SwitchStorageDefinitionsTest(NVRsTest.java:1837)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SwitchStorageDefinitionsTest -->
  <testcase name="ViewsFilterTest" time="3.099" classname="testcases.ViewsTest"/>
  <testcase name="WebPagesFilterTest" time="5.433" classname="testcases.WebPagesTest"/>
  <testcase name="addViewTest" time="25.067" classname="testcases.MonitoringView"/>
  <testcase name="ChangeNameExitCancelUnsavedChangesExitCancelTest" time="41.751" classname="testcases.NVRsElementsTest">
    <failure type="org.openqa.selenium.JavascriptException" message="JavaScript error (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: &amp;apos;3.9.1&amp;apos;, revision: &amp;apos;63f7b50&amp;apos;, time: &amp;apos;2018-02-07T22:25:02.294Z&amp;apos;
System info: host: &amp;apos;BORSE-PC&amp;apos;, ip: &amp;apos;172.20.1.42&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_202&amp;apos;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:43342/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: c2d40a57-49f8-4f9a-832f-16027b6127c1">
      <![CDATA[org.openqa.selenium.JavascriptException: JavaScript error (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'BORSE-PC', ip: '172.20.1.42', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities {browserName: internet explorer, javascriptEnabled: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, se:ieOptions: {browserAttachTimeout: 0, elementScrollBehavior: 0, enableElementCacheCleanup: true, enablePersistentHover: true, ie.browserCommandLineSwitches: , ie.enableFullPageScreenshot: true, ie.ensureCleanSession: true, ie.fileUploadDialogTimeout: 3000, ie.forceCreateProcessApi: false, ie.forceShellWindowsApi: false, ie.useLegacyFileUploadDialogHandling: false, ignoreProtectedModeSettings: true, ignoreZoomSetting: false, initialBrowserUrl: http://localhost:43342/, nativeEvents: true, requireWindowFocus: false}, unexpectedAlertBehaviour: dismiss, unhandledPromptBehavior: dismiss, version: 11}
Session ID: c2d40a57-49f8-4f9a-832f-16027b6127c1
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:160)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:537)
at Utilities.Page.JavaScriptClick(Page.java:757)
at pageObjects.NVRsPage.AddDiscoveredNVRs(NVRsPage.java:934)
at pageObjects.NVRsPage.IfNVRIsNotExistAddIt(NVRsPage.java:955)
at testcases.NVRsElementsTest.ChangeNameExitCancelUnsavedChangesExitCancelTest(NVRsElementsTest.java:131)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ChangeNameExitCancelUnsavedChangesExitCancelTest -->
  <testcase name="RemoveNVRsByButtonTest" time="30.291" classname="testcases.NVRsTest"/>
  <testcase name="CancelDeletingViewByButton" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- CancelDeletingViewByButton -->
  <testcase name="ChangeNameExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeNameExitAndCancelTest -->
  <testcase name="ChangeVisibilityStatusExitAndDoNotSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeVisibilityStatusExitAndDoNotSaveTest -->
  <testcase name="ChangeIPAdressExitPressSaveUnsavedChangesTest" time="0.0" classname="testcases.NVRsElementsTest">
    <ignored/>
  </testcase> <!-- ChangeIPAdressExitPressSaveUnsavedChangesTest -->
  <testcase name="ChangeNameAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeNameAndCancelTest -->
  <testcase name="ChangeRefreshTimeStatusExitAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeStatusExitAndSaveTest -->
  <testcase name="CancelRemovingViewByIconTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- CancelRemovingViewByIconTest -->
  <testcase name="AddNewWebPageInputParametersTest" time="0.0" classname="testcases.WebPagesTest">
    <ignored/>
  </testcase> <!-- AddNewWebPageInputParametersTest -->
  <testcase name="DeleteURLExitAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- DeleteURLExitAndSaveTest -->
  <testcase name="ChangeRemarksAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRemarksAndCancelTest -->
  <testcase name="ClearViewAndDragResourceTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- ClearViewAndDragResourceTest -->
  <testcase name="DragResourceTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- DragResourceTest -->
  <testcase name="AddNewViewWithoutLayoutTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- AddNewViewWithoutLayoutTest -->
  <testcase name="ChangeVisibilityStatusExitAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeVisibilityStatusExitAndSaveTest -->
  <testcase name="ChangeUserNameToValidAndCheckStatusTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- ChangeUserNameToValidAndCheckStatusTest -->
  <testcase name="ChangeRefreshTimeValueExitAndDoNotSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeValueExitAndDoNotSaveTest -->
  <testcase name="ResourceFilterTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- ResourceFilterTest -->
  <testcase name="ChangeURLExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeURLExitAndCancelTest -->
  <testcase name="ChangeRefreshTimeStatusAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeStatusAndCancelTest -->
  <testcase name="DeleteViewByButtonTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- DeleteViewByButtonTest -->
  <testcase name="AddNewWebPageExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- AddNewWebPageExitAndCancelTest -->
  <testcase name="ChangeRefreshTimeValueBySpinnerAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeValueBySpinnerAndSaveTest -->
  <testcase name="ChangeRemarksExitAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRemarksExitAndSaveTest -->
  <testcase name="ChangeRefreshTimeValueExitAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeValueExitAndSaveTest -->
  <testcase name="ChangeRefreshTimeValueExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeValueExitAndCancelTest -->
  <testcase name="AddNewWebPageInputURLExitAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- AddNewWebPageInputURLExitAndSaveTest -->
  <testcase name="AddNewWebPageAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- AddNewWebPageAndCancelTest -->
  <testcase name="ChangeVisibilityStatusExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeVisibilityStatusExitAndCancelTest -->
  <testcase name="DeleteOneNVRAndCheckPropertiesPanelTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- DeleteOneNVRAndCheckPropertiesPanelTest -->
  <testcase name="DeleteURLExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- DeleteURLExitAndCancelTest -->
  <testcase name="ChangeStorageDefinitionsTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- ChangeStorageDefinitionsTest -->
  <testcase name="ChangeURLExitAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeURLExitAndSaveTest -->
  <testcase name="ChangeRefreshTimeValueAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeValueAndCancelTest -->
  <testcase name="ChangeStorageDefinitionsAndPressCancelTest" time="0.0" classname="testcases.NVRsElementsTest">
    <ignored/>
  </testcase> <!-- ChangeStorageDefinitionsAndPressCancelTest -->
  <testcase name="IncreasePortInAddNVRManualyWindowTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- IncreasePortInAddNVRManualyWindowTest -->
  <testcase name="DeleteViewByIconTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- DeleteViewByIconTest -->
  <testcase name="DeleteCameraAndCheckInViewTest" time="0.0" classname="testcases.ViewsTest">
    <ignored/>
  </testcase> <!-- DeleteCameraAndCheckInViewTest -->
  <testcase name="AddNewWebPageExitAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- AddNewWebPageExitAndSaveTest -->
  <testcase name="ChangePortToSecuredAndCheckStatusTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- ChangePortToSecuredAndCheckStatusTest -->
  <testcase name="AddNVRWithSecuredPortAndSwitchOffSecuredStatusTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- AddNVRWithSecuredPortAndSwitchOffSecuredStatusTest -->
  <testcase name="ChangeNameExitAndSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeNameExitAndSaveTest -->
  <testcase name="DeleteCameraAndCheckInViewTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- DeleteCameraAndCheckInViewTest -->
  <testcase name="changelayoutTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- changelayoutTest -->
  <testcase name="ChangeRefreshTimeValueBySpinnerExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeValueBySpinnerExitAndCancelTest -->
  <testcase name="ChangeRemarksExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRemarksExitAndCancelTest -->
  <testcase name="DeleteURLAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- DeleteURLAndCancelTest -->
  <testcase name="AddNewWebPageInputURLExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- AddNewWebPageInputURLExitAndCancelTest -->
  <testcase name="DragGroupToGroupTest" time="0.0" classname="testcases.GroupsHierarchyTest">
    <ignored/>
  </testcase> <!-- DragGroupToGroupTest -->
  <testcase name="ViewsFilterTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- ViewsFilterTest -->
  <testcase name="ChangeRefreshTimeValueBySpinnerAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeValueBySpinnerAndCancelTest -->
  <testcase name="ChangeURLExitAndDoNotSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeURLExitAndDoNotSaveTest -->
  <testcase name="DeleteURLExitAndDoNotSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- DeleteURLExitAndDoNotSaveTest -->
  <testcase name="ChangeStorageDefinitionsExitCancelUnsavedChangesTest" time="0.0" classname="testcases.NVRsElementsTest">
    <ignored/>
  </testcase> <!-- ChangeStorageDefinitionsExitCancelUnsavedChangesTest -->
  <testcase name="AddNewWebPageExitAndDoNotSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- AddNewWebPageExitAndDoNotSaveTest -->
  <testcase name="SwitchOffStorageDefinitionsTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- SwitchOffStorageDefinitionsTest -->
  <testcase name="ChangeLayoutTest" time="0.0" classname="testcases.MonitoringView">
    <ignored/>
  </testcase> <!-- ChangeLayoutTest -->
  <testcase name="ChangePortBySpinnersTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- ChangePortBySpinnersTest -->
  <testcase name="DecreasePortInAddNVRManualyWindowTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- DecreasePortInAddNVRManualyWindowTest -->
  <testcase name="ChangeRefreshTimeStatusExitAndDoNoTSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeStatusExitAndDoNoTSaveTest -->
  <testcase name="ChangePortToValidAndCheckStatusTest" time="0.0" classname="testcases.NVRsTest">
    <ignored/>
  </testcase> <!-- ChangePortToValidAndCheckStatusTest -->
  <testcase name="ChangeRefreshTimeStatusExitAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeStatusExitAndCancelTest -->
  <testcase name="ChangeNameExitAndDoNotSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeNameExitAndDoNotSaveTest -->
  <testcase name="ChangeVisibilityStatusAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeVisibilityStatusAndCancelTest -->
  <testcase name="ChangeRefreshTimeValueBySpinnerExitAndDoNotSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRefreshTimeValueBySpinnerExitAndDoNotSaveTest -->
  <testcase name="AddNewWebPageInputURLAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- AddNewWebPageInputURLAndCancelTest -->
  <testcase name="ChangeRemarksExitAndDoNotSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeRemarksExitAndDoNotSaveTest -->
  <testcase name="ChangeURLAndCancelTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- ChangeURLAndCancelTest -->
  <testcase name="AddNewWebPageInputURLExitAndDoNotSaveTest" time="0.0" classname="testcases.WebPagesElementsTest">
    <ignored/>
  </testcase> <!-- AddNewWebPageInputURLExitAndDoNotSaveTest -->
  <testcase name="IncreasePortAndPressCancel" time="0.0" classname="testcases.NVRsElementsTest">
    <ignored/>
  </testcase> <!-- IncreasePortAndPressCancel -->
</testsuite> <!-- Default test -->
